pkg/controller/status/status.go

```diff
@@ -751,7 +759,7 @@ func (m *statusManager) clearDegraded() {
 	defer m.lock.Unlock()
 
 	conditions := []operator.TigeraStatusCondition{
-		{Type: operator.ComponentDegraded, Status: operator.ConditionFalse},
+		{Type: operator.ComponentDegraded, Status: operator.ConditionFalse, Reason: NotAvailable, Message: "Not Available"},
```

Comment:
clearDegraded() is called to reset the condition to the most basic state, which means that it is not degraded.
The reason should reflect that most basic state. NotAvailable means almost the opposite of that. I suppose we could add a parameter to the `SetDegraded(...)` method for the message and the reason and make sure that there is always a degraded message and reason whenever this method is called.
@tmjd  What do you think?